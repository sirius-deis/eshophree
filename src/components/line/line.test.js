import {render, screen} from '@testing-library/react'
import Line from './line'

describe('Line', () => {
  it('should match snapshot', () => {
    const {container} = render(<Line width={100} />)
    expect(container).toMatchSnapshot()
  })
});