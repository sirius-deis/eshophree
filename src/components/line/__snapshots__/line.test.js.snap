// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB iPXChy"
    width="100"
  />
</div>
`;
